tests/hmm/pars.py

```diff
@@ -70,7 +70,7 @@
             "ttjets_sl", "ttjets_dl",
             "dy_m105_160_amc_01j", "dy_m105_160_vbf_amc_01j",
             "dy_m105_160_amc_2j", "dy_m105_160_vbf_amc_2j",
-            "www","wwz","wzz","zzz",
+            #"www","wwz","wzz","zzz",
```

Comment:
Hi @nanlu06, is there some reason for which these samples are commented out?

Reply:
because this bkg is zero in the last few bins of DNN, and very small overall, to prevent issues to the likelihood models, I think it is OK to remove them. Also, I am thinking to remove a few negligible systematic uncertainties with the size of the uncertainty less than 1%.

Reply:
what about combining the vvv and vv processes and ttbar+single top like we discussed? OR do we still remove them to simplify things

Reply:
OK, sorry I forgot. let me change it now.